File: 08.03/horsemeet.c
#include<stdio.h>
/*
 * Desafio E
 * Horsemeet Gym - 101954G
 */

int main(){
    int a, b, c, d;

    scanf("%d %d",&a, &b);
    scanf("%d %d",&c,&d);

    if((a+b)%2==(c+d)%2){
        printf("black\n");
    }else{
        printf("white\n");
    }
}

File: março/08.03/horsemeet.py
#!/usr/bin/python
'''
desafio E
diferenca menor que 10^-6 - empate draw
Horsemeet Gym - 101954G

'''
def main():
    a = input()
    b = input()
    c = input()
    d = input()

    if((a+b)%2 == (c+d)%2):
        print('black')
    else:
        print('white')


main()

